docs/source/contents/4_draft_mapping.rst

```diff
@@ -329,15 +333,17 @@ ISBNs
 Use Case
 ^^^^^^^^
 
-International Standard Book Numbers are present as identifier values in the Children's Defense Fund collection. As these
-identifiers have meaning outside of the context of UTK and might be used by patrons in a search to find these materials,
-it is important that we continue to support a unique field for these values.
+International Standard Book Numbers are present as identifier values in the Children's Defense Fund collection. UTK
+currently has a specific Solr field for publication identifiers (ISBNs and ISSNs) so that these identifiers can be
+displayed and searched for separately: utk_mods_publication_identifier_ms.
 
 Justification
 ^^^^^^^^^^^^^
 
-Properties for ISBN values are established in DBpedia and the Standard Identifiers Scheme. Because we give preference to
-core ontologies rather than library specific ones, we selected `dbo:issn`.
+As these identifiers have meaning outside of the context of UTK and might be used by patrons in a search to find these materials,
+it is important that we continue to support a unique field for these values.Properties for ISBN values are established
```

Comment:
"...values. Properties ..." missing a space.

Reply:
Thanks - I've addressed this.